src/main/java/net/sf/jabref/logic/importer/fetcher/ArXiv.java

```diff
@@ -107,6 +107,10 @@ public ArXiv(ImportFormatPreferences importFormatPreferences) {
     }
 
     private Optional<ArXivEntry> searchForEntryById(String identifier) throws FetcherException {
+        identifier = identifier.replace("arxiv:", "");
+        identifier = identifier.replace("arXiv:", "");
```

Comment:
I would vote for a ArXiv class, similar to the DOI one which can handle such a case.